### ISSUE TITLE
Question about inference

### ISSUE THREAD

**Issue by SY-Xuan** [google-research/pix2seq#15]
During inference, the token (5-th) that is corresponded to the object class may be classified to the coordinates. In the other hand, the token that is corresponded to the coordinates still has chance to be classified to the class of the object. How to deal with such situation? Thanks a lot.

**Comment by chentingpc** [comment]
you can offset logits before sampling if you want to disable certain predictions (e.g. set -1e9 for logits that correspond to class/coordinates). but we find that just free sampling is fine.

**Comment by SY-Xuan** [comment]
Thanks for your kindly replying!
